infraestructure/config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type Configuration interface {
+	Load() error
+}
+
+type configuration struct {
+	ConfData confData
+	Wd       string
+}
+
+type confData struct {
+	Csv         string `json:"csv"`
+	PokeBaseUrl string `json:"pokeBaseUrl"`
```

Comment:
Same as my previous comment
```suggestion
	CSV         string `json:"csv"`
	PokeBaseURL string `json:"pokeBaseUrl"`
```